Slim/Web/ImageProxy.pm

```diff
@@ -72,12 +72,7 @@ original artwork is of considerable size, where the bandwidth to download the im
 =cut
 
 use strict;
-use Digest::MD5;
-use File::Spec::Functions qw(catdir);
-use File::Slurp ();
-use HTTP::Status qw(
-	RC_MOVED_PERMANENTLY
-);
```

Comment:
This one is still being used

Reply:
It was being collapsed into one line because of the whitespace tidying.